src/draft.ts

```diff
@@ -9,6 +9,7 @@ export interface Draft {
     isFolderMapped(path: string) : boolean;
     packs() : Promise<string[] | undefined>;
     invoke(args: string) : Promise<ShellResult>;
+    invokeAsync(command : string) : Promise<ShellResult>;
```

Comment:
`invoke` is already async (returns a Promise).  Do we truly need a new method here?  If so, can we choose a name that distinguishes it more clearly?  (And yes, this interface is inconsistent with the `kubectl` one, but that's historical, and `kubectl` will likely move towards purely async methods and drop the suffix.)